### DISCUSSION TITLE
React usage

### DISCUSSION THREAD

**Discussion by AustanBrown** [nadchif/html-duration-picker.js#155] [Q&A]
In one of the previous discussions, there was a question of whether the library can be used in React.js. I had integrated it with a React project before viewing the discussion. I noticed that the onChange event is only triggered if the time is modified via the keyboard, not with the arrows that the control provides.

Is this due to the fact that React is not yet fully supported with the library? If not, is there a custom DOM event that one can use on the arrows?

**Comment by dev-hardev** [comment]
You should consider using https://reactdatepicker.com/ for react-specific projects. 
This library has almost every possible configuration for date-time inputs. @AustanBrown
